recipes-kernel/linux/files/android-drivers.cfg

```diff
@@ -0,0 +1,8 @@
+CONFIG_ANDROID=y
```

Comment:
Was this commit supposed to be part of this PR?

Reply:
in theory I wanted to open a new PR for it.